contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -629,16 +629,14 @@ func validateLeadershipTransition(desired, current string) {
 }
 
 // hacked from https://github.com/GoogleCloudPlatform/kubernetes/blob/release-0.14/cmd/kube-apiserver/app/server.go
-func newEtcd(etcdConfigFile string, etcdServerList []string) (client *etcd.Client, err error) {
-	if etcdConfigFile != "" {
-		client, err = etcd.NewClientFromFile(etcdConfigFile)
-	} else {
-		client = etcd.NewClient(etcdServerList)
+func newEtcd(etcdServerList []string) (etcd.Client, error) {
```

Comment:
were you going to update `newEtcd` in `kube-apiserver/app/server.go`? also, what are we going to do about `etcdConfigFile`? that's currently the only way to connect to a secured etcd